vcluster/configure/vcluster-yaml/sync/to-host/core/pods/README.mdx

```diff
@@ -8,6 +8,13 @@ description: Configuration for ...
 import DisableResourceWarning from '../../../../../../_fragments/hints/warning-do-not-disable-resource.mdx'
 import Patches from '../../../../../../_fragments/patches.mdx'
 import SyncPods from '../../../../../../_partials/config/sync/toHost/pods.mdx'
+import Admonition from '@theme/Admonition';
+
+<Admonition type="info" title="Config change">
+  The configuration for the `rewriteHosts.initContainer.image` was changed in vcluster version `0.27.0`.
+  Instead of a string the field now accepts an object that allows more granular control over the registry, repository and tag of the image used.
+  See [config](#pods-rewriteHosts-initContainer) for details.
+</Admonition>
```

Comment:
Message is fine, think it's fairly clear this is breaking change. Will leave up to @joaniekube to decide if admonition is right place for this, vs a section lower down.